src/App.js

```diff
@@ -1,26 +1,78 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
+import { request } from './api/api';
 
-export class App extends Component {
-  state = {
-    movies: data,
+export const App = () => {
+  const [movies, setMovies] = useState(data);
+  const [query, setQuery] = useState('');
+  const [film, findFilm] = useState(null);
+  const [error, findError] = useState(false);
```

Comment:
```suggestion
  const [movies, setMovies] = useState(data);
```
All those variables should be handled inside the `FindMovie` component.
All you need here - is `movies` (to pass it to the `MoviesList`) and function `addFilm` (to add new movies to the `movies` aray. Ths function you will pass to the `FindMovie` component to handle search there)